internal/grader/reject.go

```diff
@@ -45,10 +45,48 @@ func (this *RejectWindowMax) fullString(now timestamp.Timestamp) string {
 		nextTime.SafeMessage(), deltaString)
 }
 
-func checkForRejection(assignment *model.Assignment, submissionPath string, user string, message string) (RejectReason, error) {
+type RejectLateWithoutAllow struct {
```

Comment:
Change name to "RejectLate".
This could be used for more in the future, and I don't want to keep it too specific.